mlcomp/parallelm/pipeline/executor.py

```diff
@@ -103,13 +103,22 @@ def all_component_dependencies(self, lang):
         accumulated_py_deps = set()
 
         comps_desc_list = components_desc.ComponentsDesc(pipeline=self.pipeline,
-                                                         comp_root_path=self._comp_root_path).load()
+                                                         comp_root_path=self._comp_root_path).load(extended=True)
         for comp_desc in comps_desc_list:
             if comp_desc[json_fields.PIPELINE_LANGUAGE_FIELD] == lang:
                 deps = comp_desc.get(json_fields.COMPONENT_DESC_PYTHON_DEPS, None)
                 if deps:
                     accumulated_py_deps.update(deps)
 
+                # for Python fetch deps from requirements.txt file
+                if lang == ComponentLanguage.PYTHON:
+                    req_file = os.path.join(comp_desc[json_fields.COMPONENT_DESC_ROOT_PATH_FIELD], "requirements.txt")
```

Comment:
Don't we need to make sure to include it with the component's files when we scan the files for a given component? Is it required to be placed it in the `includeGlobPattern`?

Reply:
yes, for now I require to place it into include patterns, but I don't mind to always include it.